ledger-service/http-json/src/itlib/scala/http/AbstractHttpServiceIntegrationTestFuns.scala

```diff
@@ -284,25 +283,36 @@ trait AbstractHttpServiceIntegrationTestFuns
   implicit protected final class `Future JsValue functions`[A](
       private val self: Future[(A, JsValue)]
   ) {
-    def parseResponse[Result: JsonReader]: Future[(A, domain.SyncResponse[Result])] =
-      self.map(_ map (decode1[domain.SyncResponse, Result](_).fold(e => fail(e.shows), identity)))
+    def parseResponse[Result: JsonReader]: Future[domain.SyncResponse[Result]] =
+      self.map { case (status, jsv) =>
+        val r = decode1[domain.SyncResponse, Result](jsv).fold(e => fail(e.shows), identity)
+        r.status should ===(status)
+        r
+      }
```

Comment:
This is the focus of this PR. We thought we might have endpoints with different HTTP status and "status" field in the JSON response, so returned both here, but in practice (at least for all our tests) we don't do that. So we can test that it's really redundant every time here, and discard.

Reply:
nice cleanup!